lightly/api/api_workflow_client.py

```diff
@@ -73,12 +73,18 @@ class ApiWorkflowClient(_UploadEmbeddingsMixin,
             but used by a workflow, the newest embedding is taken by default
     """
 
-    def __init__(self, token: str, dataset_id: str = None, embedding_id: str = None):
+    def __init__(self, token: str = None, dataset_id: str = None, embedding_id: str = None):
 
         self.check_version_compatibility()
 
         configuration = Configuration()
         configuration.host = getenv('LIGHTLY_SERVER_LOCATION', 'https://api.lightly.ai')
+        if token is None:
+            token = getenv('LIGHTLY_TOKEN', None)
+            if token is None:
+                raise ValueError(f"Either provide a 'token' argument or export"
```

Comment:
FYI. There is no need to use an f-string in python if you don't include variables. 

https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python